cfn-resources/project/cmd/resource/resource.go

```diff
@@ -7,15 +7,27 @@ import (
 
 	"github.com/aws-cloudformation/cloudformation-cli-go-plugin/cfn/handler"
 	"github.com/aws/aws-sdk-go/service/cloudformation"
-	"github.com/mongodb/mongodbatlas-cloudformation-resources/project/cmd/validation"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util"
-	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/constants"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/logger"
 	progressevents "github.com/mongodb/mongodbatlas-cloudformation-resources/util/progressevent"
 	"github.com/mongodb/mongodbatlas-cloudformation-resources/util/validator"
 	"go.mongodb.org/atlas/mongodbatlas"
 )
 
+const (
+	PubKey = "ApiKeys.PublicKey"
+	PvtKey = "ApiKeys.PrivateKey"
+	OrgID  = "OrgId"
+	Name   = "Name"
+	ID     = "Id"
```

Comment:
Can we create something similar to https://github.com/mongodb/mongodb-atlas-cli/blob/master/internal/flag/flags.go with all the const definitions so that can be reused by multiple resources?